RackApp/app.rb

```diff
@@ -0,0 +1,71 @@
+require "erb"
```

Comment:
It would be nice to use rubocop to check for code styling ;)
You can make another commit for it

Example, rubocop will tell you to not forget about `# frozen_string_literal: true`

Some good reads why we use such tools: https://rubystyle.guide/

Also about the [*magic comment*](url) `frozen_string_literal: true`

https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/

This might go probably in Ruby 3.0 default https://bugs.ruby-lang.org/issues/11473 (still in discussion)